turbinia/workers/__init__.py

```diff
@@ -1039,10 +1039,14 @@ def run_wrapper(self, evidence):
 
     log.debug(f'Task {self.name:s} {self.id:s} awaiting execution')
     failure_message = None
+    worker_name = platform.node()
     try:
       evidence = evidence_decode(evidence)
       self.result = self.setup(evidence)
       self.update_task_status(self, 'queued')
+      task_key = self.state_manager.redis_client.build_key_name('task', self.id)
+      self.state_manager.redis_client.set_attribute(
+          task_key, 'worker_name', json.dumps(worker_name))
```

Comment:
That seems reasonable for calling ```update_task```. And, yes it requires the json serialization.

Reply:
Did some testing on this and calling update_task here will result in some Redis data clobbering. I suspect this is because the TurbiniaTask object in the worker differs in some attributes to the 'stub' TurbiniaTask that the server uses to create the Redis metadata (needs more testing). We can leave this as is and come back to it later. This will likely be a non-issue if we create a new 'task_state' attribute that stores the celery state, and we can then just save that in Redis through the task manager instead of doing it in the worker.